instat/dlgNewDataFrame.vb

```diff
@@ -52,45 +59,124 @@ Public Class dlgNewDataFrame
         ucrNewDFName.SetSaveTypeAsDataFrame()
         ucrNewDFName.SetLabelText("New Data Frame Name:")
         ucrNewDFName.SetPrefix("data")
-    End Sub
 
-    ' updating controls doesn't update the function
-    ' Sheet name is not on the dialog.
+        'ucrRdoOptions
+        ucrPnlDataFrame.AddRadioButton(rdoConstruct)
+        ucrPnlDataFrame.AddRadioButton(rdoCommand)
+        ucrPnlDataFrame.AddRadioButton(rdoRandom)
+        ucrPnlDataFrame.AddRadioButton(rdoEmpty)
+
+        'TODO:Providing conditions here may not be easy, find a way to do this properly!
+        ucrPnlDataFrame.AddFunctionNamesCondition(rdoConstruct, "data.frame")
+        ucrPnlDataFrame.AddFunctionNamesCondition(rdoCommand, ucrInputCommand.GetText())

```

Comment:
@shadrackkibet I don't find this line helping much, The command in `ucrInputCommand ` will keep changing.
`ucrPnlDataFrame.AddFunctionNamesCondition(rdoCommand, ucrInputCommand.GetText())`

Reply:
I agree, you can just comment this out at the moment.